makefile

```diff
@@ -93,7 +94,7 @@ dep-add: build/containerpilot_build
 # run 'GOOS=darwin make tools' if you're installing on MacOS
 ## set up local dev environment
 tools:
-	@go version | grep 1.8 || (echo 'go1.8 not installed'; exit 1)
+	@go version | grep 1.8 || (echo 'go1.8 should be installed')
```

Comment:
Yes! I've been testing locally under 1.9rc2 and would like to continue moving my local Go version. My hope is that our build process enforces our [Go version](https://github.com/joyent/containerpilot/blob/master/Dockerfile#L1) target and not our local toolchain. I think a developer warning should be enough here.